x/profile/keeper/msg_add_validator_operator_address.go

```diff
@@ -23,19 +22,6 @@ func (k msgServer) AddValidatorOperatorAddress(
 		Description:       types.ValidatorDescription{},
 	}
 
-	// remove the operator address from previous address
-	if previousValByOpAddr, found := k.GetValidatorByOperatorAddress(ctx, opAddr); found {
-		lastValidator, found := k.GetValidator(ctx, previousValByOpAddr.ValidatorAddress)
-		if !found {
-			return nil, spnerrors.Criticalf(
-				"validator should exist for operator address %s",
-				previousValByOpAddr.ValidatorAddress,
-			)
-		}
-		lastValidator = lastValidator.RemoveValidatorOperatorAddress(opAddr)
```

Comment:
We can maybe remove `RemoveValidatorOperatorAddress` definition if we no longer use it